controllers/metric_controller.go

```diff
@@ -337,7 +336,7 @@ func (r *MetricReconciler) resolveLegacyURL(ctx context.Context, t *optimizev1be
 	}
 
 	// Look for the special placeholder hostname that indicates we should look up a service
-	if u.Hostname() != optimizev1alpha1.LegacyHostnamePlaceholder {
+	if u.Hostname() != "redskyops.dev" {
```

Comment:
Does it make sense to introduce a constant for this so changing it to something else down the road is a lot easier - perhaps just in the one spot?

Reply:
I mentioned it in the description, the use of redskyops.dev as a placeholder is going away, I really just need something more obvious to make sure I can grep out where it is still being used.